.git-blame-ignore-revs

```diff
@@ -1 +1,4 @@
+# Initial formatting of astroid
 add5f7b8eba427de9d39caae864bbc6dc37ef980
+# Apply black on doc/conf.py
+2dd9027054db541871713ef1cb1ae89513d05555
```

Comment:
```suggestion
2dd9027054db541871713ef1cb1ae89513d05555
# Black's 2024 style
396f01a15d1cb0351b33654acdeedde64f537a0c
```

Reply:
We missed other big formatting change tbh, but maybe not worth it to track them down.

Reply:
* | 396f01a1 - [style] Apply black 2024's style (#2367) (3 months ago) <pre-commit-ci[bot]>

* | 0942d8fa - Bump black from 23.1a1 to 23.1.0 (#2011) (1 year, 3 months ago) <dependabot[bot]>

* | 380e8861 - [pre-commit] Trying out black 23.1a1 (#1909) (1 year, 4 months ago) <Pierre Sassoulas>

* | 344454c8 - Bump black from 22.10.0 to 22.12.0 (#1900) (1 year, 5 months ago) <dependabot[bot]>

* | b0220bdd - Bump black from 22.8.0 to 22.10.0 (#1825) (1 year, 7 months ago) <dependabot[bot]>

* | 731d0515 - Apply black on the mechanize brain (1 year, 7 months ago) <Pierre Sassoulas>

* | 3a0b104e - Bump black from 22.6.0 to 22.8.0 (1 year, 8 months ago) <dependabot[bot]>

* | 88ccd6fd - Bump black from 22.3.0 to 22.6.0 (#1672) (1 year, 10 months ago) <dependabot[bot]>

| * 264357f5 - Bump black from 22.1.0 to 22.3.0 (#1496) (2 years, 1 month ago) <dependabot[bot]>

* | 3ecb96dc - Bump black from 22.1.0 to 22.3.0 (#1496) (2 years, 1 month ago) <dependabot[bot]>

* 2dd90270 - Apply black on doc/conf.py too (2 years, 10 months ago) <Pierre Sassoulas>

| * 7409caf9 - Reformats according to black (3 years, 2 months ago) <hippo91>

* 40373681 - Fix black issues (3 years, 2 months ago) <Marc Mueller>

* b20cc9c5 - Update black version - tox (3 years, 2 months ago) <Marc Mueller>

* 0d8ed3f8 - pylint + black formatting (3 years, 3 months ago) <hippo91>

| * | | | | | 01c678ad - Formatting according to black (3 years, 3 months ago) <hippo91>

| * | | | | f1613f46 - Reformated according to black (3 years, 4 months ago) <hippo91>

| | * | | | 3554cab5 - Formatting according to black (3 years, 4 months ago) <hippo91>

| * | | | | a324fbdf - Formatting according to black (3 years, 4 months ago) <hippo91>

| * | | | dbe56fcc - Formatting according to black (3 years, 5 months ago) <hippo91>

* 18c99c4e - Reformatting according to black (4 years, 5 months ago) <hippo91>

| * | | | 4ab338ed - Refomat according to black (5 years ago) <hippo91>

| | * eeae266d - Reformat according to black (5 years ago) <hippo91>

* 3d58a62c - Let formatting be handled by black (6 years ago) <Claudiu Popa>

* c6581e98 - Add black step in tox.ini and Travis, as well as the badge (6 years ago) <Claudiu Popa>

Reply:
Yeah when I started with one hash I wasn't trying to get em all 😉